odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -35,8 +35,10 @@ def nl2br_enclose(string, enclosure_tag='div'):
     """ Like nl2br, but returns enclosed Markup allowing to better manipulate
     trusted and untrusted content. New lines added by use are trusted, other
     content is escaped. """
-    converted = nl2br(escape(string))
-    return Markup(f'<{enclosure_tag}>{converted}</{enclosure_tag}>')
+    return Markup('<{enclosure_tag}>{converted}</{enclosure_tag}>').format(
```

Comment:
Why this commit ? the diff seems useless to me (we don't really care about false positive of semgrep, right ?), also I rather prefer a f-string than the ugly format())

Reply:
It doesn't hurt and shows good practice IMO.

Reply:
My bad

Reply:
Stupid people would probably gladly have an explanation of what "Avoids false positive / review site for semgrep" means ^^